src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] for ages 0, 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for ages 14, 14', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
```

Comment:
The test description 'should return [0, 0] for ages 14, 14' is correct as per the task requirements, and the expected output `[0, 0]` is also correct.